firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -25,10 +25,13 @@ def setUp(self):
         self.prefs.set_pref(self.PREF_LOCATION_BAR_SUGGEST, 0)
 
     def tearDown(self):
-        FirefoxTestCase.tearDown(self)
+        # Close the autocomplete results
+        self.browser.navbar.locationbar.autocomplete_results.close()
 
-        self.prefs.reset_pref(self.PREF_LOCATION_BAR_SUGGEST)
+        FirefoxTestCase.tearDown(self)
 
+    # TODO: Replace with places module method when available
+    # https://bugzilla.mozilla.org/show_bug.cgi?id=1121731
```

Comment:
The module will land soon, so you can change all those lines and make use of the appropriate methods. I will add the dependency to the bug.